src/test/java/com/endava/internship/collections/StudentSetTest.java

```diff
@@ -0,0 +1,217 @@
+package com.endava.internship.collections;
+
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+import java.util.*;
+import java.time.LocalDate;
+
+public class StudentSetTest {
+    private StudentSet set;
+
+    private final static Student st1 = new Student("John", LocalDate.of(2000, 5, 12), "");
+
+    private final static Student st2 = new Student("David", LocalDate.of(2001, 6, 20), "");
+
+    private final static Student st3 = new Student("Dmitrii", LocalDate.of(1996, 10, 16), "");
+
+    @BeforeEach
+    void setUpStudentSet(){
+        set = new StudentSet();
```

Comment:
remove this setup method, and initialise in line set like this :  
private StudentSet set = new StudentSet();

Reply:
If I need it to be cleared or created new set after each test?

Reply:
check section example of section 2 Default Test Lifecycle

Reply:
on every test method, a new Set object is created